features/machine/OCP24363.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Machine taints
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  Scenario: [MAO] Reconciling machine taints with nodes
+    Given I have an IPI deployment
+    Then I store the last provisioned machine in the :machine clipboard
+
+   Then I run the :get admin command with:
+      | resource      | machine                 |
+      | resource_name | <%= cb.machine %>       |
+      | o             | yaml                    |
+   And I save the output to file>machines_to_taint.yml
+
+   Given I have the machine yaml file machines_to_taint.yml and I add the taints [{\"effect\"=>\"\", \"key\"=>\"\", \"value\"=>\"\"}]
```

Comment:
@miyadav I worry that this will get messy as the argument gets bigger... How about reading in a file instead or doing this escaping?

Reply:
thanks , I am trying what you suggested , made changes , will try to implement step definition for that , if you find there is easy way , please provide pointers ...